Server/src/config/envs/docker.ts

```diff
@@ -53,7 +53,7 @@ export function createDockerConfig() {
         username: 'citrine',
         password: 'citrine',
         storage: '',
-        sync: false,
+        sync: true,
```

Comment:
is this a permanent thing that should happen or just temporary for testing? if the latter, can this be reverted?